.dockerdev/Dockerfile

```diff
@@ -0,0 +1,63 @@
+ARG RUBY_VERSION
+FROM ruby:$RUBY_VERSION-slim-buster
+
+ARG PG_MAJOR
+ARG NODE_MAJOR
+ARG BUNDLER_VERSION
+ARG YARN_VERSION
+
+# Common dependencies
+RUN apt-get update -qq \
+  && DEBIAN_FRONTEND=noninteractive apt-get install -yq --no-install-recommends \
+    build-essential \
+    gnupg2 \
+    curl \
+    less \
+    vim \
```

Comment:
Do you need to install vim here since it's also in `.dockerdev/Aptfile`? This is a cool pattern that I haven't seen before though. Why not put more of the dependencies in there?

Reply:
Yeah, vim could be removed from here.

> Why not put more of the dependencies in there?

The Aptfile is meant for development tools and other deps not required for the app itself to run.

Reply:
Thanks for explaining.